lib/Components/MultisigWalletInfo.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Header, Table } from '@bpanel/bpanel-ui';
+
+import { safeEval, preventDefault, buildSingleColumnTable } from '../utilities';
+import { PLUGIN_NAMESPACE } from '../constants';
+
+import styles from '../styles';
+
+class MultisigWalletInfo extends Component {
+  constructor() {
+    super();
+  }
+
+  static get propTypes() {
+    return {
+      handleSelect: PropTypes.func
+    };
+  }
+
+  render() {
+    const { handleSelect, balance, cosigners, account, id, m, n } = this.props;
```

Comment:
these should probably all be in the propTypes getter

Reply:
Updated 👍